config.json

```diff
@@ -1295,7 +1295,13 @@
         "name": "Word Search",
         "uuid": "0125f5a7-8883-4553-a6c1-45f19544af5e",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "arrays",
+          "array-loops",
+          "array-transformations",
```

Comment:
I don't think transformations are needed/recommended to solve this, e.g. have a look at this solution: https://exercism.org/tracks/javascript/exercises/word-search/solutions/James-Deciutiis